modules/openxr/scene/openxr_hand.cpp

```diff
@@ -130,8 +157,9 @@ Skeleton3D *OpenXRHand::get_skeleton() {
 	return skeleton;
 }
 
-void OpenXRHand::_get_bones() {
-	const char *bone_names[XR_HAND_JOINT_COUNT_EXT] = {
+void OpenXRHand::_get_joint_data() {
+	// The OpenXR skeleton rig bone names ("_L" or "_R" will be appended).
+	static const char *openxr_bone_names[XR_HAND_JOINT_COUNT_EXT] = {
```

Comment:
Same nitpick, I would make this a `bone_names[SKELETON_RIG_MAX][XR_HAND_JOINT_COUNT_EXT]` array.

Reply:
Done. I also converted this to a table of `String` which prevents unnecessary type-conversion in `_get_joint_data` at runtime.

Reply:
Makes sense!